### DISCUSSION TITLE
How to store a one to many relation in Log.

### DISCUSSION THREAD

**Discussion by Sumonchandrashil** [spatie/laravel-activitylog#810] [Q&A]
I have two models. 
1) MoneyReceipt 
2) MoneyReceiptDetails
I want to log MoneyReceipt along with MoneyReceiptDetails. 
Is there any scope?

**Comment by Gummibeer** [comment]
You can't log one to many relations as there's no event or anything.
The easiest is to log changes on both models.

**Reply by Sumonchandrashil** [reply]
Thanks.
